openid-4-verifiable-presentations-1_0.md

```diff
@@ -549,11 +555,273 @@ The Wallet then validates the request as specified in OAuth 2.0 [@RFC6749].
+
+A valid VP Query is defined as a JSON-encoded object with the following
+top-level properties:
+
+`expect_credentials`:
+: REQUIRED. A non-empty array of Credential Queries as defined in (#credential_query)
+that specify the requested Verifiable Credentials.
+
+`valid_credential_sets`:
+: OPTIONAL. A non-empty array containing arrays of identifiers for elements in
+`expect_credentials` that defines which sets of Credentials may be returned.
+The identifier MAY be postfixed by `?`, indicating that
+delivery of the the respective credential is optional. For details, see
+(#vp_query_lang_processing_rules).
+
+Note: While this specification does not define additional top-level properties,
+future extensions MAY define additional properties that modify the behavior of
+the Wallet when processing the query.
+
+## Credential Query {#credential_query}
+
+A Credential Query is an object representing a request for one specific
+credential. 
+
+It contains the following properties:
+
+`id`:
+: REQUIRED. A string identifying the credential in the response and, if provided,
+the constraints in `valid_credential_sets`.
+
+`format`:
+: REQUIRED. A string that specifies the format of the requested
+Verifiable Credential. Valid Credential format identifier values are defined in
+Appendix A of [@!OpenID.VCI]. The value of this property MUST be one of the
+supported Credential formats as defined in the Wallet's metadata.
+
+`expect_meta`: 
+: OPTIONAL. An object defining additional properties requested by the Verifier that
+apply to the metadata and validity data of the credential. The properties of
+this object are defined per format in (#format_specific_properties). If omitted,
+no specific restrictions are placed on the metadata or validity of the requested
+Credential.
+
+`expect_claims`:
+: OPTIONAL. A non-empty array of objects as defined in (#claims_query) that specifies
+claims in the requested Credentials.
+
+`valid_claim_sets`:
+: OPTIONAL. A non-empty array containing arrays of identifiers for elements in
+`expect_claims`. The identifier MAY be postfixed by `?`, indicating that
+delivery of the the respective claim is OPTIONAL.  
+
+## Claims Query {#claims_query}
+
+Each entry in `expect_claims` MUST be an object with the following properties:
+
+`id`:
+: REQUIRED if `valid_claim_sets` is present; OPTIONAL otherwise. A string
+identifying the particular claim. The value MUST be a non-empty string
+consisting of alphanumeric, underscore (`_`) or hyphen (`-`) characters.
+Within the particular `expected_claims` array, the same `id` MUST NOT
+be present more than once.
+
+`path`:
+: REQUIRED if the credential format uses a JSON-based claims structure; MUST NOT
+be present otherwise. The value MUST be a claims path pointer that specifies the path to the claim
+within the Verifiable Credential, as defined in (#claims_path_pointer).
+
+`namespace`:
+: REQUIRED if the credential format is based on ISO 18013-5; MUST NOT be present otherwise. 
```

Comment:
```suggestion
: REQUIRED if the Credential Format is based on mdoc format defined in ISO 18013-5; MUST NOT be present otherwise. 
```
As I read it 18013-5 defines the generic mdoc format and the mDL. I would make it clear here this is about mdoc and not mDL